src/gf_nf4.h

```diff
@@ -122,8 +122,10 @@ bool NF4<T>::check_n(unsigned n)
     return (n <= sizeof(T) / 4);
 }
 
+#ifdef NTTEC_USE_SIMD
```

Comment:
Maybe we should group all the SIMD specialization declaration in order to have a single `ifdef` block in this file. That would be more readable/easier to maintain.